File: src/main/java/com/pubnub/api/models/server/SubscribeEnvelope.java
package com.pubnub.api.models.server;

import com.google.gson.annotations.SerializedName;
import lombok.Getter;

import java.util.List;

@Getter
public class SubscribeEnvelope {

    @SerializedName("m")
    private List<SubscribeMessage> messages;

    @SerializedName("t")
    private SubscribeMetadata metadata;

}

File: src/main/kotlin/com/pubnub/api/models/server/SubscribeEnvelope.kt
package com.pubnub.api.models.server

import com.google.gson.annotations.SerializedName

class SubscribeEnvelope(

    @SerializedName("m")
    internal val messages: List<SubscribeMessage>,

    @SerializedName("t")
    internal val metadata: SubscribeMetaData
)
